plugin/PlayAssetDelivery/src/androidTest/java/com/google/play/core/godot/ExampleInstrumentedTest.java

```diff
@@ -0,0 +1,42 @@
+/*
```

Comment:
I wonder if we should move these test files to play/core/godot/assetpacks since that's where our eventual unit and instrumented tests will live.